Resources/Prototypes/SS220/Entities/Structures/Doors/Airlocks/airlocks.yml

```
@@ -21,3 +21,27 @@
     department: Security
   - type: Wires
     layoutId: AirlockSecurity
+
+- type: entity
+  parent: Airlock
+  id: AirlockLawSecurity
+  suffix: Security/Lawyer
+  components:
+  - type: Sprite
+    sprite: SS220/Structures/Doors/Airlocks/Standart/law_sec.rsi
+  - type: PaintableAirlock
+    department: Law
```

```suggestion
    department: Law
```
Тут закон, а ниже секурити, в чем разница?

---

хз)

---

Может тогда унифицируем под охрану только?
чтобы не запутаться и не было таких вопросов более